src/main/java/core/basesyntax/FileEditor.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+public class FileEditor {
+
+    public void start() {
+        try (Scanner scanner = new Scanner(System.in)) {
+            Command command = new Command(scanner);
+            boolean bool = true;
+            while (bool) {
+                System.out.println("Write command or text: ");
+                String[] input = scanner.nextLine().split(" ");
+                String path = input.length > 1 ? input[1] : "";
+                String fileName = input.length > 2 ? input[2] : "";
+                switch (getCommand(input)) {
+                    case 0:
+                        System.out.println("Your input is empty!");
+                        break;
+                    case 1:
+                        command.createFile(path, fileName);
+                        break;
+                    case 2:
+                        command.readFile(path, fileName);
+                        break;
+                    case 3:
+                        command.getInfo(path, fileName);
+                        break;
+                    case 4:
+                        command.callHelp(path);
+                        break;
+                    case 5:
+                        command.callHelp();
+                        break;
+                    case 6:
+                        bool = false;
+                        break;
```

Comment:
let's give more informative name for this variable

Reply:
done